.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,12 @@
+blank_issues_enabled: true
+contact_links:
```

Comment:
Should we remove "contact links" as we don't have dissuasions for rules yet?

Reply:
Hey @pierssinclairssw ,

Do we want to keep the config.yml for this, or remove it and keep the other templates?